Apps/WebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -43,6 +44,10 @@ import UserRating from "@/models/userRating";
 import VaccinationStatus from "@/models/vaccinationStatus";
 import { RootState } from "@/store/types";
 
+export interface IHttpDelegateService {
```

Comment:
This initialize signature is declared 21 times in this file without variation, hence the generalized interface. I would like to apply it to all of the interfaces, but can be done after delivery as this doesn't not directly impact the delivery of this work such as the PatientData refactor yesterday.

Att: @sslaws, @ytqsl